plugins/home/backstage-plugin-home-rss/package.json

```diff
@@ -35,7 +35,7 @@
     "@backstage/config": "^1.0.8",
     "@backstage/core-components": "^0.13.2",
     "@backstage/core-plugin-api": "^1.5.2",
-    "@backstage/plugin-home": "^0.5.3",
```

Comment:
This modification should contain the related changes to a yarn.lock file.

Reply:
I don't know if I did something wrong. But I executed `yarn install` and the yarn.lock didn't change at all. I even tried removing the node_modules inside this plugin

Reply:
That's no worries. It could be that this dep is already included in the with the same versioning